conformance/operator_test.go

```diff
@@ -933,13 +1034,51 @@ func createSriovPolicy(sriovDevice string, testNode string, numVfs int, resource
 	}, 10*time.Minute, time.Second).Should(Equal(int64(numVfs)))
 }
 
-func createTestPod(node string, networks []string) *k8sv1.Pod {
+func createUnschedulableTestPod(node string, networks []string, resourceName string) {
 	podDefinition := pod.RedefineWithNodeSelector(
 		pod.DefineWithNetworks(networks),
 		node,
 	)
 	createdPod, err := clients.Pods(namespaces.Test).Create(podDefinition)
+	Eventually(func() k8sv1.PodPhase {
```

Comment:
You should use `Consistently` here, as this will success even for pods that are temporarly in Pending phase and then go to the running phase.

Reply:
Done